3-UsingAPIs/Week1/homework/ex4-pokerDiceAll.js

```diff
@@ -27,9 +27,9 @@ exercise file.
 const rollDie = require('../../helpers/pokerDiceRoller');
 
 function rollDice() {
-  // TODO Refactor this function
   const dice = [1, 2, 3, 4, 5];
-  return rollDie(1);
+  const anArrayOfPromises = dice.map((element) => rollDie(element));
```

Comment:
The `.map()` expects a function that at minimum takes one parameter, which is an element of the array. The `rollDie()` function meets this requirement. Therefore, you can optionally shorten this to:

```suggestion
  const anArrayOfPromises = dice.map(rollDie);
```